### PR TITLE
Add wgpu

### LINKED ISSUE #7636: Request: wgpu
### Brand Name

wgpu

### Website

https://wgpu.rs/

### Popularity Metric

[Global Rank 1,029,919](https://www.similarweb.com/website/wgpu.rs/#overview)
[5.3k+ GitHub stars](https://github.com/gfx-rs/wgpu)

### Official Resources for Icon and Color

https://github.com/gfx-rs/wgpu/blob/master/logo.png

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by elliotwutingfeng**
image

**Issue:** closes #7636

**GitHub stars:**  10k

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

The homepage's hero background color is the same as the bulma CSS primary color, which is turquoise `#40E0D0`.

### CODE REVIEW

**Comment by adamrusted** [comment]
Do we think adding in the gaps between colour changes would help this icon, @simple-icons/icon-review?

**Comment by LitoMore** [comment]
We can ask the thier official people since they have GitHub.

**Comment by PeterShaggyNoble** [comment]
> We can ask the thier official people since they have GitHub.

Pinging @jimblandy & @cwfitzgerald from the @gfx-rs team for their feedback here.

I myself would be in favour of cut-outs.

**Comment by service-paradis** [comment]
I'm also in favor of cut-outs. Without any answer, I think we can go forward and adjust if we get any feedback in the future. Do you want to give this a try @elliotwutingfeng?

**Comment by cwfitzgerald** [comment]
I'm no graphics designer, so do whatever you see fit that preserves the intent of the logo :)

**Review by PratyushJayachandran — APPROVED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -17126,6 +17126,11 @@
             "hex": "4E49EE",
             "source": "https://github.com/wez/wezterm/blob/fe78b5821fc106c1061f4c1cc454ff01e74bf97d/assets/icon/wezterm-icon.svg"
         },
+        {
+            "title": "wgpu",
+            "hex": "40E0D0",
+            "source": "https://wgpu.rs/logo.min.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:17132]:
```suggestion
            "source": "https://wgpu.rs"
```

**Review by adamrusted — APPROVED** [review]
Quick change to the URL used, but other than that this is good to go.

**Pull request merged**